crates/ironrdp-connector/src/lib.rs

```diff
@@ -48,7 +49,7 @@ pub struct BitmapConfig {
 #[derive(Debug, Clone)]
 pub enum Credentials {
     UsernamePassword { username: String, password: String },
-    SmartCard { pin: String },
+    SmartCard { pin: String, config: Option<SmartCardIdentity> },
```

Comment:
issue: This requires types from another crate into the public API of `ironrdp-connector`.

suggestion: Re-export the required types from this module using `pub use` or expose a wrapper struct that we internally convert into the sspi type later.

Reply:
I created wrapper structure, seems to be cleaner this way, `cargo xtask ci` is green locally for me